Quaver.Shared/Screens/Options/OptionsMenu.cs

```diff
@@ -362,6 +362,7 @@ private void CreateSections()
                         new OptionsItemKeybind(containerRect, "Increase Scroll Speed", ConfigManager.KeyIncreaseScrollSpeed),
                         new OptionsItemKeybind(containerRect, "Decrease Map Offset", ConfigManager.KeyDecreaseMapOffset),
                         new OptionsItemKeybind(containerRect, "Increase Map Offset", ConfigManager.KeyIncreaseMapOffset),
+                        new OptionsItemKeybind(containerRect, "Toggle Playtest Autoplay", ConfigManager.KeyTogglePlaytestAutoplay),
```

Comment:
This would fit better in the editor section.

Reply:
This is also in map preview though

Reply:
Oh never mind, it shouldn't be in gameplay anyway

Reply:
it can also fit in `song selection` in that case